Makefile

```diff
@@ -214,23 +219,28 @@ github-token.txt:
 	echo $(GITHUB_TOKEN) >> github-token.txt
 
 
-ifneq ($(strip $(GITHUB_ACTION)),)
-$(DOCKER_DEVKIT_PHONY_FILE): github-token.txt
-endif
+.PHONY: buildx
+buildx:
+buildx:
+	 docker buildx create --driver=docker-container --use
+	 docker run --privileged --rm tonistiigi/binfmt --install all
+
+
+$(DOCKER_DEVKIT_PHONY_FILE): github-token.txt buildx
 $(DOCKER_DEVKIT_PHONY_FILE): Dockerfile.devkit install-envsubst
 		docker buildx build \
 		$(BUILD_FLAGS) \
-		--tag "$(DOCKER_DEVKIT_IMG)" \
-		--load \
 		$(REPO_ROOT_DIR) \
-	&& touch $(DOCKER_DEVKIT_PHONY_FILE)
+	&& docker load --input /tmp/img.tar && rm /tmp/img.tar && touch $(DOCKER_DEVKIT_PHONY_FILE) && docker images
```

Comment:
Nice. We will have to make similar changes to make it work with goreleaser. I am wondering if we should look into refactor these targets and directly use goreleaser. ironically gorleaser target runs in Devkit container. 🙄

Reply:
I think the additional changes in the release-tag action should make it so that each devkit is built beforehand and would use a cache

Reply:
Even a multi stage build? This would still benefit from caching.